### ISSUE TITLE
Sprint 3: Legally Blond Llaso Apso  💁🏼‍♀️🐕

### ISSUE THREAD

**Issue by alisonjoseph** [carbon-design-system/carbon#7751]
## February 8th - February 19th

**Team issues**
- Grid discussion, planning 
- Release planning

**Andrea**
- [ ] Sketch pictogram/plugin work - Epic: https://github.com/carbon-design-system/carbon/issues/6485
  - [x] Plugin layer color duplication - PR https://github.com/carbon-design-system/carbon/pull/7762
- [ ] Continue work on https://github.com/carbon-design-system/carbon/issues/6938 with @joshblack 
  - [x] Fix combobox WCAG violation. - PR https://github.com/carbon-design-system/carbon/pull/7777
- [ ] Accessibility Audit continued:
   - [x] Fix Overflow Menu a11y bug - PR https://github.com/carbon-design-system/carbon/pull/7812
   - [x] Fix Tooltip Menu a11y bug - PR https://github.com/carbon-design-system/carbon/pull/7821
   - [x] Fix Pictograms Img Alt a11y violation - PR: https://github.com/carbon-design-system/carbon-website/pull/2149
   - [x] Fix Breadcrumbs - header a11y violation - PR: https://github.com/carbon-design-system/carbon-website/pull/2150

**Andrew**
- [ ] Sketch pictogram/plugin work - Epic: https://github.com/carbon-design-system/carbon/issues/6485
  - [x] Plugin layer color duplication - PR https://github.com/carbon-design-system/carbon/pull/7762
- [x] fix(FilterableMultiSelect): extend input outline to trigger button area #7722 #7535
- [x] fix(multi-select): remove left padding on filterable input after selection #7740 #7735
- [x] fix(Toggle): support labels for small toggles #7752 #7750
- [x] feat(ComboBox): support warning state #7733 #7685
- [x] fix(toggle): adjust disabled toggle styles in data table #7753 #7351
- [x] fix(PasswordInput): add missing button class #7799 #7765
- [x] fix: update listbox titleText proptypes to match Dropdown #7785 #7779
- [x] feat(Modal): implement support for 3 button modals #7775
- [x] fix(OverflowMenu): resolve child.props NPE #7838 #7837

**Anna**
- [x] Grid - sync w/ security
- [ ] Grid - layout create examples
- [x] Color token v11 discussion 
- [x] v11 proposals

**DA**
- [x] Continue work on https://github.com/carbon-design-system/carbon/issues/7200
  - [ ] Pair with one developer to help onboard with JAWS
- [ ] https://github.com/carbon-design-system/carbon/issues/6840, pair with @joshblack and @andreancardona to see if overlap with #6938 

**Jan**
- [x] Concept work with Matt, Jeannie
- [ ] Pattern guidance review
- [x] Clarification of [Button usage guidance](https://github.com/carbon-design-system/carbon-website/issues/2095) @janchild 

**Jeannie**
- [ ] Concept work with Matt, Jan
- [ ] Pattern guidance review

**Jillian**
- [ ] [Develop case study content](https://github.com/carbon-design-system/carbon-website/issues/2071) for the Cloud transition to Carbon 10  

**John**
- [ ] Create add-on package for animated icons #7346 

**Josefina**
- [x] https://github.com/carbon-design-system/carbon/issues/7476
- [ ] https://github.com/carbon-design-system/carbon-website/issues/2125
- [ ] Pair with @tay1orjones on Grid work

**Josh**
- [x] Continue work on https://github.com/carbon-design-system/carbon/issues/6938 with @andreancardona 
  - [x] Fix combobox WCAG violation. - PR https://github.com/carbon-design-system/carbon/pull/7777
- [x] https://github.com/carbon-design-system/carbon/issues/7713, pair with @tw15egan and @laurenmrice 
- [x] Work with @dakahn and @tw15egan over MultiSelect Filterable consolidation in https://github.com/carbon-design-system/carbon/issues/7331 and filter into #6938 work with Andrea
- [x] Establish early release process for early releases for sponsor teams, work with @tay1orjones https://github.com/carbon-design-system/carbon/issues/7778
- [x] Figure out fix for CVE flagged for telemetry package

**Lauren**
- [ ] [Context menu](https://github.com/carbon-design-system/carbon/pull/7350) content guidance (in progress)
- [ ] Status of Treeview component (guidance/kit/code) (in progress)
- [ ] Popover disclosure (in progress)
- [x] Sketch plugin support

**Taylor**
- Continue Grid work from last sprint, pair with @jnm2377 
- [ ] Dive into File uploader issues
  - [ ] https://github.com/carbon-design-system/carbon/issues/6321
  - [ ] https://github.com/carbon-design-system/carbon/issues/5894
- [x] Establish early release process for early releases for sponsor teams, work with @joshblack  
- [x] #7647
- [x] https://github.com/carbon-design-system/carbon-website/pull/2136
- [x] https://github.com/carbon-design-system/carbon/issues/7731

**Thy**
- [x] Comms images
- [ ] Status icons

**TJ**
- Work with Adir and team to help them contribute BIDI support/fixes for our components
- [ ] https://github.com/carbon-design-system/carbon/issues/7713, pair with @joshblack and @laurenmrice on Popover
- [x] https://github.com/carbon-design-system/carbon/issues/7724
- [x] https://github.com/carbon-design-system/carbon/issues/7476
- [x] https://github.com/carbon-design-system/carbon/issues/6636
- [ ] https://github.com/carbon-design-system/carbon/issues/7757

**Tracey**
- Component docs
  - [ ] Tabs

**Comment by joshblack** [comment]
## Development proposal

**Andrea**

- [ ] Sketch pictogram/plugin work - Epic: https://github.com/carbon-design-system/carbon/issues/6485
- [ ] Continue work on https://github.com/carbon-design-system/carbon/issues/6938 with @joshblack 

**Andrew**

- [ ] Sketch pictogram/plugin work - Epic: https://github.com/carbon-design-system/carbon/issues/6485


**DA**

- [ ] Continue work on https://github.com/carbon-design-system/carbon/issues/7200
  - [ ] Pair with one developer to help onboard with JAWS
- [ ] https://github.com/carbon-design-system/carbon/issues/6840, pair with @joshblack and @andreancardona to see if overlap with #6938 


**Josefina**

- [ ] https://github.com/carbon-design-system/carbon/issues/7476
- [ ] https://github.com/carbon-design-system/carbon-website/issues/2125
- [ ] Pair with @tay1orjones on Grid work

**Josh**

- [ ] Continue work on https://github.com/carbon-design-system/carbon/issues/6938 with @andreancardona 
- [ ] https://github.com/carbon-design-system/carbon/issues/7713, pair with @tw15egan and @laurenmrice 
- [ ] Work with @dakahn and @tw15egan over MultiSelect Filterable consolidation in https://github.com/carbon-design-system/carbon/issues/7331 and filter into #6938 work with Andrea
- [ ] Establish early release process for early releases for sponsor teams, work with @tay1orjones 
- [ ] Figure out fix for CVE flagged for telemetry package

**Taylor**

- Continue Grid work from last sprint, pair with @jnm2377 
- [ ] Dive into File uploader issues
  - [ ] https://github.com/carbon-design-system/carbon/issues/6321
  - [ ] https://github.com/carbon-design-system/carbon/issues/5894
- [ ] Establish early release process for early releases for sponsor teams, work with @joshblack  
- [ ] https://github.com/carbon-design-system/carbon/issues/7647

**TJ**

- [ ] https://github.com/carbon-design-system/carbon/issues/7713, pair with @joshblack and @laurenmrice on Popover
- [ ] https://github.com/carbon-design-system/carbon/issues/7724
- [ ] https://github.com/carbon-design-system/carbon/issues/7476
- [ ] https://github.com/carbon-design-system/carbon/issues/6636

**Comment by aagonzales** [comment]
Design epics on deck:
- [Community pattern migration](https://github.com/carbon-design-system/carbon-website/issues/2086)
- [Remaining component usage docs](https://github.com/carbon-design-system/carbon-website/issues/2085)

Design support issues:
- Selected buttons https://github.com/carbon-design-system/carbon/issues/7757
- Content switch 3:1 contrast issue https://github.com/carbon-design-system/carbon/issues/6745
- Button guidance, v11 suggestions (new secondary btn)
